archetypes/archetype_definition_qby_msp.json

```diff
@@ -4,7 +4,8 @@
             "QBY-Deploy-VM-Backup",
             "QBY-Deploy-DomainJoin",
             "Deploy-VM-Monitoring",
-            "QBY-Deploy-LAW-Agent"            
+            "QBY-Deploy-LAW-Agent",
+            "QBY-Deploy-Require-Tags"
```

Comment:
I think we should add this policy for root and not only MSP, as these tags should also be required for the customer

Reply:
Changed as suggested.